spec/bedrock_runtime/client_spec.rb

```diff
@@ -8,7 +8,8 @@
     described_class.new(
       region: ENV.fetch('AWS_REGION', nil),
       access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID', nil),
-      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY', nil)
+      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY', nil),
+      profile: ENV.fetch('AWS_PROFILE', nil)
```

Comment:
Please add a new test scenario where the client instantiation is done without the `profile` so both cases are covered.